subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/DefaultWatchingVirtualFileSystem.java

```diff
@@ -108,32 +107,32 @@ public void stopWatching() {
         }
 
         AtomicInteger count = new AtomicInteger();
-        AtomicBoolean overflow = new AtomicBoolean();
+        AtomicBoolean unknownEventEncountered = new AtomicBoolean();
         try {
             watchRegistry.stopWatching(new FileWatcherRegistry.ChangeHandler() {
                 @Override
                 public void handleChange(FileWatcherRegistry.Type type, Path path) {
                     count.incrementAndGet();
+                    LOGGER.debug("Handling VFS change {} {}", type, path);
```

Comment:
Can we move this to info logging while the feature is in the early phase? Or directly log it to the daemon log. WDYT?

Running with debug output on a bigger project is very painful.